src/frontend/js/widgets/SyllabusCourseRunsList/components/SyllabusSimpleCourseRunsList/index.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+import { CourseRunFactory } from 'utils/test/factories/richie';
+import { SyllabusSimpleCourseRunsList } from '.';
+
+jest.mock('widgets/SyllabusCourseRunsList/components/CourseRunItem', () => ({
+  __esModule: true,
+  default: () => 'CourseRunItem',
+}));
+
+jest.mock('widgets/SyllabusCourseRunsList/components/CourseRunItemWithEnrollment', () => ({
+  __esModule: true,
+  default: () => 'CourseRunItemWithEnrollment',
+}));
+
+describe('SyllabusSimpleCourseRunsList', () => {
+  it('should render CourseRunItemWithEnrollment when checkEnrollment is true', () => {
+    const courseRun = CourseRunFactory().one();
+
+    render(<SyllabusSimpleCourseRunsList courseRuns={[courseRun]} checkEnrollment />);
+
+    screen.getByText('CourseRunItemWithEnrollment');
```

Comment:
I don't find this text.
May be use a `data-testid` in order to check that we use the right component ?

Reply:
I mocked components `CourseRunItem` and `CourseRunItemWithEnrollment` at the top of the test.